trickster/endpoints/internal.py

```diff
@@ -47,8 +47,8 @@ def create_route(route: InputRoute, mocked_router: Router = Depends(get_router))
         new_route.validate_existing_response_validator_combinations()
         mocked_router.add_route(new_route)
         return new_route
-    except ValidationError as e:
-        raise HTTPException(status_code=400, detail=f'Failed validation: {e}') from e
+    except pydantic.ValidationError as exception:
```

Comment:
Can you please search the project for ` as e:`? I think we usually use just `e` instead of `exception`. If we haven't use it in this project yet, I'm ok with calling it `exception`. Just make sure it's consistent